test/models/stake_pool_test.rb

```diff
@@ -1,7 +1,53 @@
 require "test_helper"
 
 class StakePoolTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    @stake_pool = create(:stake_pool,
+                         network: "testnet",
+                         name: "TestName",
+                         authority: "authority")
+
+    @validator1 = create(:validator, :with_score, name: "Validator", account: "Account")
+    @validator2 = create(:validator, :with_score, name: "Validator", account: "Account2")
+
+    [5_000_000_000_000, 4_000_000_000_000].each do |stake_value|
+      create(:stake_account,
+             network: "testnet",
+             validator: @validator1,
+             stake_pool: @stake_pool,
+             active_stake: stake_value,
+             delegated_stake: stake_value,
+             staker: "staker1")
+    end
+
+    [2_000_000_000_000, 1_000_000_000_000].each do |stake_value|
+      create(:stake_account,
+             network: "testnet",
+             validator: @validator2,
+             stake_pool: @stake_pool,
+             active_stake: stake_value,
+             delegated_stake: stake_value,
+             staker: "staker1")
+    end
+  end
+
+  test "validators_count" do
```

Comment:
please add more detailed description to tests, eg. "#validators_count returns number of validators associated with the stake pool" or something like that